package.json

```diff
@@ -1,4 +1,8 @@
 {
+	"name": "refined-twitter",
+	"private": true,
+	"repository": "https://github.com/sindresorhus/refined-twitter",
+	"license": "MIT",
```

Comment:
Why add this? It's not needed since it's not going to be published on npm.

Reply:
npm was complaining

```
npm WARN refined-twitter No repository field.
npm WARN refined-twitter No license field.
```

Reply:
Just ignore npm on that. They don't require it at all so not sure why they would warn on it.